core/src/Streamly/Internal/Data/MutArray/Type.hs

```diff
@@ -2292,6 +2293,7 @@ pinnedWriteNUnsafe :: forall m a. (MonadIO m, Unbox a)
     => Int -> Fold m a (MutArray a)
 pinnedWriteNUnsafe = unsafeCreateOf'
 
+-- XXX Use "IO" instead of "m"
```

Comment:
In the alloc function.

Important: leave a blank line between this comment and the haddock comments below, otherwise the doctests won't run.